### DISCUSSION TITLE
Audio Switcher features (switch device by clicking icon/keyboard shortcuts & change icon to show current device)

### DISCUSSION THREAD

**Discussion by gee3000** [File-New-Project/EarTrumpet#1615] [Ideas]
I've been using Audio Switcher (https://github.com/xenolightning/AudioSwitcher_v1 - similar to SoundSwitch) for many years and recently started using EarTrumpet.

Problem is I now have 3 audio related icons in my system tray (including the system icon which can't be hidden in Windows 11), and I thought it would be ideal to have features of both combined in one app.

- [ ] 1. switch output device (cycle through) by clicking the system tray icon (middle-click?) and/or via keyboard shortcuts
- [ ] 2. change system tray icon to show the current device

I appreciate both apps serve different purposes so this request may be outside the scope of this project, but the missing features seem like fairly simple quality of life improvements.

**Comment by HurricAngel** [comment]
I came from Soundswitch and the one thing I cannot give up is the systray icon that reflects the current selected device. Other than this EarTrumpet is superior in every way :)

**Comment by Arimodu** [comment]
> Problem is I now have 3 audio related icons in my system tray (including the system icon which can't be hidden in Windows 11), and I thought it would be ideal to have features of both combined in one app.

There is a [Windhawk](https://windhawk.net/) mod that allows you to hide the windows 11 volume icon like so: 
<img width="285" height="86" alt="image" src="https://github.com/user-attachments/assets/d1499f1d-3313-4385-a783-e58aa564403c" />

Its called:
<img width="403" height="213" alt="image" src="https://github.com/user-attachments/assets/78a99dcc-86d7-492b-b288-35aaa1742789" />
